xrpl/openapi-codegen/models/issued_currency.py

```diff
@@ -0,0 +1,25 @@
+"""Model for IssuedCurrency."""
+
+from dataclasses import dataclass
+from typing import Optional
+from xrpl.models.base_model import BaseModel
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class IssuedCurrency(BaseModel):
+    currency: Optional[str] = None
```

Comment:
Shouldn't all types have a description?

Reply:
Yes, we will fix after revising the spec